core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -676,6 +652,45 @@ private[deploy] class Master(
     logInfo(f"Recovery complete in ${timeTakenNs / 1000000000d}%.3fs - resuming operations!")
   }
 
+  private[master] def handleRegisterWorker(
```

Comment:
You mean this method's parameters change?

Reply:
But its code is still the same right? (copied from existing code)

Reply:
Yes, the code is the same. I change the parameter from `RegisterWorker` class to field parameters to avoid side-effects.